streaming/index.js

```diff
@@ -575,7 +590,7 @@ const startWorker = (workerId) => {
 
   app.get('/api/v1/streaming/direct', (req, res) => {
     const channel = `timeline:direct:${req.accountId}`;
-    streamFrom(channel, req, streamToHttp(req, res), streamHttpEnd(req, subscriptionHeartbeat(channel)), true);
```

Comment:
I believe this was a bug.

Reply:
Y

Reply:
how so?

Reply:
Why does the direct timeline need filtering (the last `true` argument)? It's already pre-filtered

Reply:
By the way I didn't mean it was a bug with a visible effect, because filtering only works on `update` events, but I believe that argument was misplaced.

Reply:
Hmmm is language filtered on the Ruby side? I don't think it is. Otherwise, everything else should be redundant, yeah

Reply:
Language filters on DMs? Never been a thing, is not supposed to be a thing.

Reply:
Apparently, it was, for `update`s, but this DMs have long been only `conversation`s anyway